docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-immutable-server-proc.adoc

```diff
@@ -0,0 +1,50 @@
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc process -n <project-name> -f <template-path>/rhpam70-prod-immutable-kieserver.yaml -v ATTRIBUTE=value -V ATTRIBUTE=value... | oc create -n <project-name> -f -
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URL for {KIE_SERVER}.
+** *KIE Server Certificate Name* (`KIE_SERVER_HTTPS_NAME`): the name of the certificate in the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Keystore Password* (`KIE_SERVER_HTTPS_PASSWORD`): the password for the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server ID* (`KIE_SERVER_ID`): the name of the server template on the {CENTRAL} that the {KIE_SERVER} is to join. Set this parameter to any value. If you do not set it and then the {KIE_SERVER} pod is restarted or scaled, each new instance of the server might join a new template. 
+** *KIE Server Container Deployment* (`KIE_SERVER_CONTAINER_DEPLOYMENT`): the identifying information of the decision service (KJAR file) that is built from your source. The format is: `<containerId>=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, for example: `containerId=groupId:artifactId:version|c2=g2:a2:v2`. The Maven build process must produce all these files from the source in the Git repository.
+** *Git Repository URL* (`SOURCE_REPOSITORY_URL`): the URL for the Git repository that contains the source for your decision service. If you want to use the Git repository that is built into your authoring {CENTRAL}, use the URL: ???
+** *Git Reference* (`SOURCE_REPOSITORY_REF`): the branch in the Git repository
+** *Context Directory* (`CONTEXT_DIR`): the path to the source within the project downloaded from the Git repository
+** *Artefact Directory* (`ARTIFACT_DIR`): the path within the project that contains the required binary files (KJAR files and any other necessary files) after a successful Maven build. Normally this directory is the target directory of the build. However, you can provide prebuilt binaries in this directory in the Git repository
+** *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`): the namespace where the image streams are available. If the image streams were already available in your OpenShift environment (see <<imagestreams-file-install-proc>>), the namespace is `openshift`. If you have installed the image streams file, the namespace is the name of the OpenShift project.
++
+You can also set other parameters as necessary.
```

Comment:
Maybe worth mentioning an optional external maven repository, that will often but not always be needed, and host the dependencies of the deployed processes.